src/node/jsonRpc/Methods/SendTransaction.js

```diff
@@ -0,0 +1,65 @@
+import {authenticatedMethod, RpcMethod} from './../../../jsonRpc';
+import {isObject, isString, isNumber} from 'lodash';
+
+/**
+ * Creates and sign a transaction based on the provided parameters
+ */
+class SendTransaction extends RpcMethod
+{
+    constructor(name, oTransactionsManager) {
+        super(name);
+
+        this._oTransactionsManager = oTransactionsManager;
+    }
+
+    async getHandler(args) {
+        if (args.length !== 1 || isObject(args[0]) === false)
+        {
+            throw new Error('Params must contain exactly one entry and that entry must be an object');
+        }
+
+        const fromAddress = args[0]['from'];
+        const toAddress   = args[0].to;
+        const value       = args[0].value;
+        const fee         = args[0].fee;
+        const password    = args[0].password;
+
+        if (isString(fromAddress) === false)
+        {
+            throw new Error('From address is invalid. Only string values are supported.');
+        }
+
+        if (isString(toAddress) === false)
+        {
+            throw new Error('To address is invalid. Only string value is supported.');
+        }
+
+        if (isNumber(value) === false || value <= 0)
+        {
+            throw new Error('Value is invalid. Only numerical values greater than 0 are supported.');
+        }
+
+        if (typeof fee !== 'undefined' && (isNumber(fee) === false || fee <= 0))
+        {
+            throw new Error('Fee is invalid. Only numerical values greater than 0 are supported.');
+        }
+
+        try
+        {
+            const aResponse = await this._oTransactionsManager.wizard.createTransactionSimple( fromAddress, toAddress, value, fee, undefined, password);
```

Comment:
This works, as in the transaction is added successfully in the Pending queue, the amount is substracted from the wallet`s balance, but the transaction is not ever processed.

Need feedback on this.

Reply:
The legacy API uses the same method to create transactions - https://github.com/WebDollar/Node-WebDollar/blob/master/src/node/sockets/node-server/API/Node-API-Private.js#L74

That might be an issue with the transaction propagation or node setup.